### PR TITLE
fix(sdk): added support of buyer tokens with different environments

### PR DESCRIPTION
**Pull request by xuelianhan007**
## Description

Added support of buyer tokens with environment signature so that the buyer tokens will be separated by different environments.
The configurations alteration pull request is as following:
https://github.com/mycloudnexus/kraken-deployment-config/pull/31
## Related Issue

https://github.com/mycloudnexus/kraken/issues/35
## How This Has Been Tested
These patch has been verified with integration test in localhost.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] CI/CD or documentation update (changes to CI/CD pipeline or documentation)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

File: kraken-app/kraken-app-controller/src/main/resources/assets/product.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
     - classpath:/mef-sonata/api-specs/api-spec.quote.yaml
     - classpath:/mef-sonata/api-specs/api-spec.product.offering.yaml
     - classpath:/mef-sonata/api-targets/api-target.order.uni.add.yaml
-    - classpath:/mef-sonata/api-targets/api-target.order.eline.add.yam
+    - classpath:/mef-sonata/api-targets/api-target.order.eline.add.yaml
     - classpath:/mef-sonata/api-targets/api-target.order.uni.add.yaml
     - classpath:/mef-sonata/api-targets/api-target.order.eline.add.yaml
```
Comment (kuangxiang20240501) [inline kraken-app/kraken-app-controller/src/main/resources/assets/product.yaml:23]: classpath:/mef-sonata/api-targets/api-target.order.eline.add.yaml   duplicate content, would u can  delete  one of them

Comment (xuelianhan007) [reply kraken-app/kraken-app-controller/src/main/resources/assets/product.yaml]: Thanks. The duplicated lines have been deleted.

**Review by kuangxiang20240501 — DISMISSED** [review]
LTGM

**Review by zhiyaoPCCW — COMMENTED** [review]
LGTM, @DaveXiong please help to review it.

**Review by DaveXiong — APPROVED** [review]
LGTM

**Pull request merged**